countdown.js

```diff
@@ -0,0 +1,5 @@
+///count down homework
+
+for (let count =10; count >= 0; count--) {
+  console.log(count)
+}
```

Comment:
make sure to put a space after the assignment operator and a semicolon after the console log

```
for (let count = 10; count >= 0; count--) {
  console.log(count);
}
```